express/src/controllers/sign-in.ts

```diff
@@ -133,3 +133,7 @@ export const processEnterMobileForm = async function (req: Request, res: Respons
         active: "sign-in"
     });
 };
+
+export const signInAgain = async function (req: Request, res: Response) {
+    res.render("you-need-to-sign-in-again.njk");
+};
```

Comment:
```suggestion
export const sessionTimeout = async function (req: Request, res: Response) {
    res.render("session-timeout.njk");
};
```

Please use the existing `session-timeout` template and delete the duplicate template.